.gitignore

```diff
@@ -6,4 +6,7 @@ npm-debug.log
 
 # IntelliJ / IDEA
 *.iml
-.idea/*
\ No newline at end of file
+.idea/*
+
+yarn-error.log
+yarn.lock
```

Comment:
My impression was that lock files needed to be checked-in, not ignored. This way, everyone is sure to get the same dependency.

Reply:
yeah your right, i just took it out temporarily. Was not sure if we wanted to go with yarn or stay with using npm.